src/pages/stacks-blockchain/transactions.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Transaction generation
+description: Guide to Stacks 2.0 transactions
+---
+
+## Introduction
+
+Transactions are the fundamental unit of execution in the Stacks blockchain. Each transaction is originated from a standard account, and is retained in the Stacks blockchain history for eternity. This guide helps you understand Stacks 2.0 transactions.
+
+If you want to jump right in and broadcast your first transaction, try this tutorial:
+
+[@page-reference | inline]
+| /stacks-blockchain/sending-tokens
+
+-> The information on this page is based on a design proposal. You can find more conceptual details in this document: [SIP 005: Blocks, Transaction, Accounts](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-005-blocks-and-transactions.md).
+
+## Lifecycle
+
+Transactions go through phases before being finally confirmed, and available for all, on the Stacks 2.0 network.
+
+![Transaction lifecycle](/images/tx-lifecycle.png)
+
+- **Generate**: Transactions are assembled according to the encoding specification.
+- **Verify and sign**: Transactions are verified to confirm if they are well-formed. Required signatures are filled in.
+- **Broadcast**: Transactions are send to a node.
+- **Register**: A miner receives transactions and adds them to the "mempool", a holding area for all the pending transactions.
+- **Process**: Miners review the mempool and select transactions for the next blocked to be mined. Depending on the transaction type, different actions can happen during this step. For example, post-conditions could be verified for a token transfer, new tokens could be minted, or an attemp to call an existing smart contract method could be made.
```

Comment:
[typo] "blocked" -> "block"
"attemp" -> "attempt"

FYI I don't think new Stacks tokens can be minted as a result of processing a transaction -- smart-contract defined tokens could be though. Might be worth clarifying that. New STX is minted via mining, but block rewards are not paid out immediately (miners have to wait 100 blocks to get rewards for a block they mined)

Reply:
I see now, you're considering "coinbase transactions" as well. Those are somewhat different in that you won't generate and broadcast them like you would other transactions.